spec/integration/runner.rb

```diff
@@ -0,0 +1,77 @@
+class Runner
+  attr_reader :pid, :output, :status
+
+  def initialize(name)
+    @script_name = name
+    @script_file = "#{@script_name}.rb"
+    @pid = nil
+    @output = nil
+    @status = nil
+    @post_spawn_wait = jruby? ? 10 : 0.5 # seconds
```

Comment:
```suggestion
    @post_spawn_wait = jruby? ? 10 : 1 # seconds
```

Might want to make the `@post_spawn_wait` a bit longer? I'm seeing this flaky behaviour where the signal is sent and trapped before `Waiting for USR1 signal..." is printed (also likely meaning, before AppSignal has started)

<img width="802" alt="Screenshot 2024-09-13 at 13 54 57" src="https://github.com/user-attachments/assets/044a8886-65f2-4cc3-b4de-9cd9c3b7b205">